onboarding-enabler-spring/src/test/java/com.ca.apiml.enable/register/ApimlEnabledRegisterToApiLayerTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package com.ca.apiml.enable.register;
+
+import com.ca.apiml.enable.EnableApiDiscovery;
+import com.ca.apiml.enable.config.EnableApiDiscoveryConfig;
+import com.ca.mfaas.eurekaservice.client.ApiMediationClient;
+import com.ca.mfaas.eurekaservice.client.config.ApiMediationServiceConfig;
+import com.ca.mfaas.exception.ServiceDefinitionException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.ConfigFileApplicationContextInitializer;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.*;
+
+
+@RunWith(SpringRunner.class)
+@EnableAutoConfiguration
+@EnableApiDiscovery
+@ContextConfiguration(initializers = ConfigFileApplicationContextInitializer.class, classes = {RegisterToApiLayer.class, EnableApiDiscoveryConfig.class})
```

Comment:
@EnableApiDiscovery is already defined. It will enable classes = {RegisterToApiLayer.class, EnableApiDiscoveryConfig.class}

Reply:
```
@EnableAutoConfiguration
@EnableApiDiscovery
@ContextConfiguration(initializers = ConfigFileApplicationContextInitializer.class)
```